okdownload/src/main/java/com/liulishuo/okdownload/core/file/MultiPointOutputStream.java

```diff
@@ -413,6 +420,8 @@ void runSync() throws IOException {
             flushProcess();
             nextParkMills = syncBufferIntervalMills;
         }
+
+        Util.d(TAG, "IO sync end");
```

Comment:
You know, here is the loop of sync process to breakpoint-store end, not I/O sync.